src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -51,7 +98,7 @@ PageWrapper.propTypes = {
 };
 
 PageWrapper.defaultProps = {
-  bbcOrigin: null,
+  bbcOrigin: '',
```

Comment:
Is there a reason this now needs to be an empty string?

Reply:
There's a proptype error if i leave it as null because null isn't of type string? I'm not sure if bbcorigin should always be required though

Reply:
Looks like `bbcOrigin` was an optional prop to `PageWrapper` prior to this work, so why is it now required within `PageWithRequestContext`?

Reply:
`bbcOrigin` needs to be an optional prop all the way down, defaulting to null, though to `getOriginContext`